src/components/LogIn.jsx

```diff
@@ -15,18 +16,25 @@ async function login(user) {
       .then((data) => data.json());
 }
 
-const LogIn = () => {
+const LogIn = (props) => {
```

Comment:
It's good practice to destructure your props in the definition, makes it easier to quickly see what the props are that are being used.

`const LogIn  ({ setLoggedIn }) => {`